frontend/lib/pages/base/home/user_menu.dart

```diff
@@ -1,31 +1,90 @@
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 
-class UserMenu extends StatelessWidget{
-
+class UserMenu extends StatelessWidget {
   const UserMenu({Key? key}) : super(key: key);
 
```

Comment:
Replace all hardcode (e.g. "David", 234, etc...) with Variables and place them above the "build"-methode